presidio-api/cmd/presidio-api/docs/readme.md

```diff
@@ -0,0 +1,25 @@
+# Swagger
```

Comment:
Looks good. Could you please reference to this readme from the main Presidio docs?

Reply:
Added a reference from the general readme